### ISSUE TITLE
[Feature Request] 3D Terrain Editor

### ISSUE THREAD

**Issue by mrmaller1905** [IGI-Research-Devs/IGI-Editor#8]
Can you make a 3D terrain editor to your program?

**Comment by haseeb-heaven** [comment]
Yes, terrain editor has already been done by Dark, so its possible yes

**Comment by mrmaller1905** [comment]
Here is the source code for 3D terrains to port into Project I.G.I ToolKit: https://github.com/hjcminus/Project-IGI-Terrain

**Comment by heaven-hm** [comment]
> Here is the source code for 3D terrains to port into Project I.G.I ToolKit: https://github.com/hjcminus/Project-IGI-Terrain

Thanks i will check this

**Comment by haseeb-heaven** [comment]
Thanks for requesting i have created now fully separate 3D Editor with Objects/AI Terrain https://github.com/jones-hm/project-igi-editor
